templates/municipal_chief/home.html.twig

```diff
@@ -86,7 +86,21 @@
                     <a href="https://chezvous.en-marche.fr/" target="_blank" class="btn btn btn--ghosting--blue">Consulter</a>
                 </footer>
             </div>
-            <div></div>
+            {% if app.user.municipalChiefManagedArea.hasJecouteAccess() %}
+                <div class="manager-overview__case">
+                    <span class="manager-overview__case--section">Communication & Mobilisation</span>
+                    <h1 class="manager-overview__case--title">J'écoute</h1>
+                    <p class="manager-overview__case--baseline">Application disponible sur
+                        <a class="text--blue--dark link--no-decor" target="_blank" href="https://play.google.com/store/apps/details?id=fr.en_marche.jecoute&hl=it">Google Play</a>
+                        et sur l'<a class="text--blue--dark link--no-decor" target="_blank" href="https://apps.apple.com/fr/app/j%C3%A9coute/id1441973895">App Store</a>.
+                    </p>
+                    <footer class="manager-overview__case--footer">
+                        <a href="{{ path('app_jecoute_municipal_chief_local_surveys_list') }}" target="_blank" class="btn btn btn--ghosting--blue">Administrer</a>
+                    </footer>
+                </div>
+                {% else %}
```

Comment:
could you remove the `else` statement? 
```twig
{% if ... %}
   ....
{% else %}
    </div></div>
{% endif %}
```
=>
```twig
{% if ... %}
   ....
{% endif %}
```

Reply:
ohh I noticed that you replace a div element `<div></div>` by two closing tags `</div></div>`. if you need to keep `else` statement maybe you should change two closing tags by `<div></div>`?

![image](https://user-images.githubusercontent.com/4582866/64255130-0d913000-cf21-11e9-9cbd-8e295b899884.png)
![image](https://user-images.githubusercontent.com/4582866/64255144-141fa780-cf21-11e9-8c19-e6ccc71ed18b.png)

Reply:
I need to have at least 2 elements in a row to have it at 50% width, I use an empty markeup when I only have one, could I do either ?

Reply:
> ohh I noticed that you replace a div element `<div></div>` by two closing tags `</div></div>`. if you need to keep `else` statement maybe you should change two closing tags by `<div></div>`?
> 
> ![image](https://user-images.githubusercontent.com/4582866/64255130-0d913000-cf21-11e9-9cbd-8e295b899884.png)
> ![image](https://user-images.githubusercontent.com/4582866/64255144-141fa780-cf21-11e9-8c19-e6ccc71ed18b.png)

ah you're right there is a `/` too much !